support/cas-server-support-redis-core/src/test/java/org/apereo/cas/redis/core/util/RedisUtilsTest.java

```diff
@@ -0,0 +1,47 @@
+package org.apereo.cas.redis.core.util;
+
+import lombok.val;
+import org.apereo.cas.authentication.CasSSLContext;
+import org.apereo.cas.configuration.model.support.redis.BaseRedisProperties;
+import org.apereo.cas.redis.core.RedisObjectFactory;
+import org.apereo.cas.util.junit.EnabledIfPortOpen;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.data.redis.core.RedisTemplate;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * This is {@link RedisUtilsTest}.
+ *
+ * @author Fireborn Z
+ * @since 6.5.0
+ */
+@Tag("Redis")
+@EnabledIfPortOpen(port = 6379)
+public class RedisUtilsTest {
```

Comment:
Please rewrite the test case to use SpringBootTest similar to other redis-based testsd.

Reply:
Changed.